index.html

```diff
@@ -4,15 +4,86 @@
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    
     <!-- Add a link to your custom CSS here -->
 
+    <link rel="stylesheet" href="css/main.css">
     <title>Responsive grid project</title>
 </head>
 
 <body>
 
     <!-- Add your markup here -->
+<main class="h-main">
 
+     <section class="h-hero">
+       <h2>
+        <input type="checkbox">Maya Nelson</h2>
+       <p>PROJECT MANAGER</p>
+     </section>
+
+     <section class="h-feature">
+        
+          <img src="maya.jpeg" alt="maya">
+          <h2>Maya</h2>
+          <h2 >Nelson</h2>
+          <button class="one"><a id="left" href="#">RESUME</a></button> <button class="two" ><a id="right" href="#">PROJECTS</a></button><br>
```

Comment:
Should these be buttons or should they be links _styled_ as buttons?

Buttons trigger an action (like in a form) and links navigate to a new location. What is happening in this interaction?

Reply:
The two buttons link to new locations to separate pages.